### PR TITLE
[PD-2003] User management - invite user

### PR DESCRIPTION
**Pull request by lemniscate613**
# Overview

This PR is mostly about integrating invitations into user management, but it should solve PD-2003 too.

These work locally, and I'd like to test them on QC:
* Add User (not in system, in system but not assigned roles, in system but assigned different roles)
* Edit User (in system)

# Test

Run Django tests:
`dkm test myjobs.tests.test_manage_users.ManageUsersTests`

Run JS tests:
`dkgg build`

Create disposable email addresses: https://www.guerrillamail.com

Update roles and users manually: https://secure.my.jobs/manage-users/

### CODE REVIEW

File: myjobs/views.py [hunk 1]
```diff
@@ -228,7 +229,7 @@ def contact_faq(request):
 
     """
     faq = FAQ.objects.filter(is_visible=True).order_by('question')
-    if not faq.count() > 0:
+    if faq.count() <= 0:
```
Comment (lemniscate613) [inline myjobs/views.py]: A PEP8 fix...

File: myjobs/views.py [hunk 2]
```diff
@@ -1017,12 +1018,27 @@ def api_get_users(request):
                                                       roles_assigned_to_user,
                                                       fields=('name'))
 
-        # TODO Do assigned roles and available roles, I use this on front end
         # Status
-        # TODO: This is NOT the same as status
-        # Waiting on email invitation work
         ctx[user.id]["status"] = user.is_verified
 
+        # Calculate pending since date
```
Comment (lemniscate613) [inline myjobs/views.py]: Used to produce this date on front-end when we invite a new user:

screen shot 2016-01-14 at 9 03 32 am

File: myjobs/views.py [hunk 3]
```diff
@@ -1112,26 +1126,15 @@ def api_create_user(request):
         if request.POST.get("user_email", ""):
             user_email = request.POST['user_email']
 
-        matching_users = User.objects.filter(email=user_email)
```
Comment (lemniscate613) [inline myjobs/views.py]: Moved this functionality down to 1134.

File: myjobs/views.py [hunk 4]
```diff
@@ -1017,12 +1018,27 @@ def api_get_users(request):
...
+        # Didn't use user.last_response because that 'denotes the last time
+        # they interacted with any email, not just invitations.' -Troy 1/13/16
+        # Instead 1) Find all invitations for this user
+        # 2) get the latest such invitation and 3) use that date
+        if user.is_verified == True:
+            lastInvitation = ''
+        else:
+            invitations = (Invitation
+                           .objects
+                           .filter(invitee_email=user.email)
```
Comment (tdphillips) [inline myjobs/views.py]: Should this also be filtering on `inviting_company`?

**Pull request merged**